### PR TITLE
Remove deprecated `ReservoirSimulationTimeSeries`

### PR DESCRIPTION
**Pull request by anders-kiaer**

### CODE REVIEW

File: webviz_examples/webviz-full-demo.yml [hunk 1]
```diff
@@ -287,18 +287,6 @@ layout:
                     visualization: statistics
```
[the remark this answers is not in the document]
Comment (rnyb) [reply webviz_examples/webviz-full-demo.yml:287]: For those using the demo as a source for copy/paste i think having some of the settings exposed can actually be quite helpful.

**Review by rnyb — COMMENTED** [review]
Looks good to me, but we may have to add more doc to the plugin itself?
https://equinor.github.io/webviz-subsurface/#/webviz-subsurface?id=simulationtimeseries

File: webviz_examples/webviz-full-demo.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (asnyv) [reply webviz_examples/webviz-full-demo.yml:287]: @rnyb I see your point. It is a trade-off. But also nice to show people that you don't really need a lot of configuration (avoid scaring people more than necessary 😉) Maybe try to add comments to those that are default values to explicitly say that they are not really needed?
Or alternatively: minimal config in the example, but a comment with a link to the right place in the documentation where the options are described (assuming you actually have it properly documented)?

[the remark this answers is not in the document]
Comment (rnyb) [reply webviz_examples/webviz-full-demo.yml:287]: That could work. A third option to consider is to have the optional settings included with default values, but commented out.

[the remark this answers is not in the document]
Comment (asnyv) [reply webviz_examples/webviz-full-demo.yml:287]: Could be :) but we are maybe into a more generic debate on how we should configure the demo? So I'll approve this one without this modification, and then we can discuss the general strategy of the demo configuration in another setting?

**Review by asnyv — APPROVED** [review]

File: webviz_examples/webviz-full-demo.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (anders-kiaer) [reply webviz_examples/webviz-full-demo.yml:287]: Created #129. I think the best of both worlds would perhaps be shorten this configuration file as much as possible, and ensure the `webviz-config-editor` reaches a level where you don't need to "copy/paste" from the demo, but instead see all available options "live" in the editor.

**Pull request merged**